docs/components/optimize/what-is-optimize.md

```diff
@@ -0,0 +1,26 @@
+---
+id: what-is-optimize
+title: What is Optimize?
+---
+
+Camunda is largely comprised of three key components:
```

Comment:
```suggestion
Camunda Cloud is built to handle three key aspects of process automation:
```
This is purely a suggestion, but let's tie this back to Camunda Cloud for sure. "Components" is an overloaded term, but I want to reserve to it mean the functional areas of Camunda Cloud - Zeebe, Modeler, Tasklist, Operate, Optimize, etc.

Reply:
Agreed 100%.